lib/tasks/remove-failed-followers.js

```diff
@@ -0,0 +1,58 @@
+/* jshint globalstrict:false, strict:false, sub: true */
+/* global print */
+exports.name = "remove-failed-followers";
+exports.group = "cleanup tasks";
+exports.args = [
+  {
+    name: "collection-integrity",
+    optional: false,
+    type: "jsonfile",
+    description: "json file created by analyze task"
+  }
+];
+exports.args_arangosh = " --server.endpoint AGENT-OR-COORDINATOR";
+exports.description = "Clears failed followers found by analyze task.";
+exports.selfTests = ["arango", "db", "leaderAgencyConnection"];
+exports.requires = "3.3.23 - 3.8.99";
+exports.info = `
+Removes failed followers found by the analyze task.
+`;
+
+exports.run = function (extra, args) {
+
+  // imports
+  const _ = require("lodash");
+  const helper = require('../helper.js');
+  const shards = helper.getValue("collection-integrity", args);
+
+  const trx = [];
+
+  for (const entry of shards.followerOnDeadServer) {
```

Comment:
Just FYI you could use object restructuring here:
`for (const {db, name, shard, currentServers, server} of shards.followerOnDeadServer) {`
This would give you `db`, `name`, `shard` etc. as direct variables, so you do not need to put `entry.db` in the code below.